Arale/AraleHeaderView.swift

```diff
@@ -85,12 +92,13 @@ public class AraleHeaderView: UIView {
         super.init(frame: frame)
     }
     
-    convenience public init(withMinHeight minHeight: CGFloat, bottomMargin: CGFloat = 0, backgroundImage: UIImage? = nil) {
+    convenience public init(withMinHeight minHeight: CGFloat, bottomMargin: CGFloat = 0, backgroundImage: UIImage? = nil, overlayView: UIView? = nil) {
         self.init(frame: CGRect.zero)
         self.minHeight = minHeight
         self.maxHeight = minHeight * 1.25
         self.bottomMargin = bottomMargin
         self.backgroundImage = backgroundImage
+        self.overlayView = overlayView
```

Comment:
as we expose the `contentView`/`overlayView` as gettable property for `AraleHeaderView` I think it's much cleaner if we let this initialized as empty `UIView`. What do you think?